CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_shopify v0.12.2
+
+## Feature: Customizable date start
+- Updated the shopify__calendar start date to include a variable `shopify__calendar_start_date`. This can be set in your dbt_project.yml. If not used, the default will start at `2019-01-01`. See the [README](https://github.com/fivetran/dbt_shopify/blob/main/README.md#Change-the-calendar-start-date) for more details. 
```

Comment:
Additionally, we should add a callout here that if customers modify this variable then they will need to run a `dbt run --full-refresh` since some downstream models that reference the `shopify__calendar` are incremental and would not capture the historical records grabbed with this new variable on subsequent incremental runs unless a full refresh in conducted.